conda_pack/__init__.py

```diff
@@ -1,4 +1,6 @@
-from . import _version
-from .core import CondaEnv, CondaPackException, File, pack
+try:
+    from ._version import version as __version__
+except ImportError:
+    __version__ = "unknown"
 
-__version__ = _version.get_versions()['version']
+from .core import CondaEnv, CondaPackException, File, pack  # noqa: F401
```

Comment:
Redundant imports or the all list are alternatives to F401

`from .core import CondaEnv as CondaEnv`,`__all__ = ["CondaEnv"]`

Reply:
Unused imports is what was being flagged.

Reply:
Those forms promote the imports to a "used import"